### ISSUE TITLE
Create an RMA from the BO to return the product/s

### ISSUE THREAD

**Issue by Jorgemgm** [thirtybees/thirtybees#535]
When a customer calls or send an email in to troubleshoot a product and we deem that it needs to be returned.

We can click the “Returns products” button in the order in Back Office, but:

The customer can’t check the status of your return by logging in to your account and clicking on “My Merchandise Returns”.

In “Return Merchandise Authorization (RMA)” will not show the “Package status” and the costumer can’t “Print out” the PDF “Return slip” and include it with your package.

**Comment by getdatakick** [comment]
I don't think this is a bug. 'Return products' button in back office order page is not tied to RMA feature at all. This is mechanism to adjust order/invoice, while RMA is a communication mechanism. 

These complement each other in some way.

I think what you want is an option to create Merchandise Return request on behalf of your customer, from back office.
